contributors/release-note-checker.md

```diff
@@ -0,0 +1,17 @@
+## Background
+As we all know, the release process of TiDB is relatively lengthy. One of the most cost time thing is the collection of release notes. We have many PRs that do not have a release note when they are merged. Therefore, a release note checker was added to some repo in CI Check the release note, please pay attention when you mention pr and review, the effect is as follows
+
+<img src="../media/release-note-1.png" alt="Release note checker" width="450"/>
+
+## Check principle and usage
+CI obtains the Release note keyword from the description, and judges that there is a release note displayed in the form of a list (internal judgment is a regular expression `Release note[\r\n]\s*[-|*].+`). If there is, it will judge success.
```

Comment:
```suggestion
CI obtains the Release note keyword from the description, and judges that there is a release note displayed in the form of a list (internal judgment is a regular expression `Release note[\r\n]\s*[-|*].+`). If there is one, it will judge as success.
```